app.js

```diff
@@ -0,0 +1,386 @@
+app.use(express.static(path.join(__dirname, "uploads")));
+
+app.use(bodyParser.urlencoded({
+    extended: true
+}));
+app.use(bodyParser.json());
+
+
+var storage = multer.diskStorage({
+    destination: function(req, file, callback) {
+        callback(null, './uploads/user');
+    },
+    filename: function(req, file, callback) {
+        var fileUniqueName = md5(Date.now());
+        callback(null,  fileUniqueName + path.extname(file.originalname));
+    }
+});
+
+var upload = multer({ storage: storage });
+
+
+//---------------------FOR DATABASE CONNECTION-----------------------------------------------//
+var connection = mysql.createConnection({
+	host     : 'localhost',
+	user     : 'root',
+	password : '',
+	database : 'fluper_user'
+});
+ 
+connection.connect(function(err){
+  if (err) {
+    console.error('error connecting: ' + err.stack);
+    return;
+  }
+  console.log('database working fine');
+});
+ 
+
+
+//-------------------------For adminlogin------------------------------------------------//
+
+app.post('/adminlogin', function(req, res) {
+	var email = req.body.email;
+	var password = req.body.password;
+
+	var sql = "SELECT * FROM `admin` WHERE `emailId`=? AND `password`=?";
+	var password = md5(password);
+	var values = [email,password];
+	console.log(values);
+
+	connection.query(sql, values, function(err, result){
+		console.log(err);
+		console.log(result);
+		if (err) {
+			console.log(err);
+		} else {
+			if(result.length>0)
+				{
+					result[0].password="";
+					var response={
+						status:1, 
+						response: result[0]
+					}
+
+					res.send(response);
+				}
+				else{
+					var response ={
+						status:0,
+
+						response:"invalid cretdential"
+
+					}
+					res.send(response);
+				}
+			}
+		});
+});
+
+//-------------------------For userlogin------------------------------------------------//
+
+app.post('/userlogin', function(req, res) {
+	var email = req.body.email;
+	var password = req.body.password;
+
+	var sql = "SELECT * FROM `user1` WHERE `emailId`=? AND `password`=?";
+	var password = md5(password);
+	var values = [email,password];
+	console.log(values);
+
+	connection.query(sql, values, function(err, result){
+		console.log(err);
+		console.log(result);
+		if (err) {
+			console.log(err);
+		} else {
+			if(result.length>0)
+				{
+					result[0].password="";
+					var response={
+						status:1, 
+						response: result[0]
+					}
+
+					res.send(response);
+				}
+				else{
+					var response ={
+						status:0,
+
+						response:"invalid cretdential"
+
+					}
+					res.send(response);
+				}
+			}
+		});
+});
+
+
+ //----------------------------For signup----------------------------------------------//
+app.post('/adminsignup', function(req, res) {
+	var name = req.body.name;
+	var email = req.body.email;
+	var password = req.body.password;
+	var contact  = req.body.contact;
+	  var text = "";
+    var possible = "123456789";
+
+    for (var i = 0; i < 4; i++) {
```

Comment:
why this loop is here

Reply:
to generate a unique 4 digit  USER_ID at the time of signup